### DISCUSSION TITLE
The ability to name/rename custom modes? Or add more custom modes.

### DISCUSSION THREAD

**Discussion by Lolcupcakes** [jordenyt/stable_diffusion_sketch#20] [Ideas]
I can never seem to remember what each custom mode does without looking back at this github or checking the settings of it. If there was an easy way to allow us to rename them, I'd appreciate it endlessly.

**Comment by jordenyt** [comment]
Thanks for being a fan of this project.  I have implemented a way to help you with the updated version.  You can now name your custom mode by "name" attribute in the JSON.  Please check it out.

**Reply by jordenyt** [reply]
![Screenshot_20240110_182909_Stable Diffusion Sketch.jpg](https://github.com/jordenyt/stable_diffusion_sketch/assets/5007252/e2f29b5e-aa1c-463c-9aea-6c748ef30f75)

This is an example on how can you name a custom mode.
